<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainUI</class>
 <widget class="QWidget" name="MainUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>626</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>680</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset theme="fcitx5-rime">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_overview">
   <item>
    <widget class="QTabWidget" name="configTab">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="shortCutTab">
      <attribute name="title">
       <string>Shortcut</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>648</width>
            <height>557</height>
           </rect>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="toggle_label">
             <property name="text">
              <string>Call-out Menu</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="fcitx::kcm::KeyListWidget" name="toggle_shortcut" native="true">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_pageup">
             <property name="text">
              <string>Page Up</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_pagedown">
             <property name="text">
              <string>Page Down</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="toggle_hfshape">
             <property name="text">
              <string>Half/Full Shape</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="toggle_ascii">
             <property name="text">
              <string>Western/Eastern</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="toggle_trasim">
             <property name="text">
              <string>Traditional/Simplified</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="fcitx::kcm::KeyListWidget" name="hotkey_pageup" native="true">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="fcitx::kcm::KeyListWidget" name="hotkey_pagedown" native="true">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="fcitx::kcm::KeyListWidget" name="hotkey_hfshape" native="true">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="fcitx::kcm::KeyListWidget" name="hotkey_ascii" native="true">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="fcitx::kcm::KeyListWidget" name="hotkey_transim" native="true">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="candidate_word_number_label">
             <property name="text">
              <string>Candidate Word Number</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="shift_l_label">
             <property name="text">
              <string>Left Shift Key</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="shift_r_label">
             <property name="text">
              <string>Right Shift Key</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QComboBox" name="shiftLeftCombo">
             <item>
              <property name="text">
               <string>Noop</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Inline ASCII</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Commit Text</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Commit Code</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Clear</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QComboBox" name="shiftRightCombo">
             <item>
              <property name="text">
               <string>Noop</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Inline ASCII</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Commit Text</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Commit Code</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Clear</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QSpinBox" name="candidateWordNumber"/>
           </item>
           <item row="9" column="0">
            <spacer name="verticalSpacer_5">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="schemaTab">
      <attribute name="title">
       <string>Schema</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="0">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="1" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Available Input Schemas:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListView" name="availIMView"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="addIMButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="icon">
              <iconset theme="go-next">
               <normaloff>.</normaloff>.</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="removeIMButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="icon">
              <iconset theme="go-previous">
               <normaloff>.</normaloff>.</iconset>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Active Input Schemas:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListView" name="currentIMView"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="moveUpButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="icon">
              <iconset theme="go-up">
               <normaloff>.</normaloff>.</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="moveDownButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="icon">
              <iconset theme="go-down">
               <normaloff>.</normaloff>.</iconset>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QLabel" name="label_4">
         <property name="toolTip">
          <string/>
         </property>
         <property name="text">
          <string>&lt;b&gt;Select Input Schema:&lt;/b&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>fcitx::kcm::KeyListWidget</class>
   <extends>QWidget</extends>
   <header>keylistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
